exp-player/addon/components/exp-video-consent.js

```diff
@@ -0,0 +1,61 @@
+import Ember from 'ember';
+import ExpFrameBaseComponent from 'exp-player/components/exp-frame-base';
+import layout from '../templates/components/exp-video-consent';
+
+export default ExpFrameBaseComponent.extend({
+    layout,
+    videoRecorder: Em.inject.service(),
+
+    didInsertElement() {
+      this.get('videoRecorder').on('onUploadDone', () => {
+        this.get('videoRecorder').destroy();
+        this.get('videoRecorder').on('onUploadDone', null);
+        this.sendAction('next')
+      });
+      this.get('videoRecorder').start('Test', this.$('#recorder'), {record: false});
```

Comment:
`'Test'` should be replaced by `'video-consent-${sessionID}'` or something similar.